File: C++/2578. Split With Minimum Sum.cpp
class Solution {
public:
    int splitNum(int n) {
        string s=to_string(n);
        sort(s.begin(),s.end());
        string a="",b="";
        for(int i=0;i<s.size();i++){
            if(i%2==0)
            a+=s[i];
            else
            b+=s[i];
        }
        return(stoi(a)+stoi(b));
    }
};

File: JAVA/2578. Split With Minimum Sum.java
class Solution {
    public int splitNum(int n) {
        String k=Integer.toString(n);
        char x[]=k.toCharArray();
        Arrays.sort(x);
        String s=new String(x);
        String a="",b="";
        for(int i=0;i<s.length();i++){
            if(i%2==0)
            a+=s.charAt(i);
            else
            b+=s.charAt(i);
        }
        return(Integer.parseInt(a)+Integer.parseInt(b));
    }
}